### PR TITLE
Update ZH_cn.jsonc

### PR DESCRIPTION
**Pull request by BillKerman**
# 🔧Modifications:
1. ➕ Add translation of terminologies and experiemental options of `topResult`, `sharedPlaylists`, `people`, `newpreset.name`... (18 items)
2.  ➕ Add translation of action `copy`, `newpreset` and `deletepreset`,
3. ➕ Add translation of language options (4 items),
4. ➕ Add translation of audio settings (3 items),
5.  ➕ Add translation of visual options and theme (6 items).

### CODE REVIEW

**Review by cryptofyre — APPROVED** [review]

File: src/i18n/zh_CN.jsonc [hunk 1]
```diff
@@ -198,15 +218,27 @@
...
+  "settings.option.general.language": "语言",
 
   // Language optgroups
+  "settings.option.general.language.main": "语言",
+  "settings.option.general.language.fun": "恶搞语言",
+  "settings.option.general.language.unsorted": "未分类",
 
   // Update Cider
   "settings.option.general.updateCider": "更新 Cider", // Button
+  "settings.option.general.updateCider.branch": "Cider更新通道", // Dropdown
```
Comment (maikirakiwi) [inline src/i18n/zh_CN.jsonc:237]: Cider 应当与后面的中文字符之间有一个空格， L238同理。

**Review by maikirakiwi — CHANGES_REQUESTED** [review]
See comments

File: src/i18n/zh_CN.jsonc [hunk 2]
```diff
@@ -198,15 +218,27 @@
...
+  "settings.option.general.updateCider.branch.description": "选择接受Cider更新的通道",
+  "settings.option.general.updateCider.branch.main": "稳定版",
```
Comment (maikirakiwi) [inline src/i18n/zh_CN.jsonc:239]: 既然都写通道了就把 “版”都去掉

**Review by maikirakiwi — CHANGES_REQUESTED** [review]

File: src/i18n/zh_CN.jsonc (hunk 2 above)
Comment (BillKerman) [reply src/i18n/zh_CN.jsonc:239]: 不好意思，誤點了request review (*꒦ິ⌓꒦ີ)

Comment (BillKerman) [reply src/i18n/zh_CN.jsonc]: ✅Done.

**Pull request merged**